core/web/presenters/evm_chain.go

```diff
@@ -34,7 +34,7 @@ func (r EVMNodeResource) GetName() string {
 // NewEVMNodeResource returns a new EVMNodeResource for node.
 func NewEVMNodeResource(node types.NodeStatus) EVMNodeResource {
 	return EVMNodeResource{NodeResource{
-		JAID:    NewJAID(node.Name),
+		JAID:    NewJAID(node.Name).GetWithPrefixedChainID(node.ChainID),
```

Comment:
It looks like nearly every case chains the calls like this. Why don't we add a new constructor instead:
```suggestion
		JAID:    NewJAIDWithChainIDPrefix(node.Name, node.ChainID),
```
Or maybe:
```suggestion
		JAID:    NewPrefixedJAID(node.ChainID, node.Name),
```

Reply:
Wanted to do this to the default constructor but not everyone uses ChainID, and wasn't sure about adding a new constructor, but sgtm